src/sagemaker/serve/builder/model_builder.py

```diff
@@ -672,3 +683,51 @@ def validate(self, model_dir: str) -> Type[bool]:
         """
 
         return get_metadata(model_dir)
+
+    def total_inference_model_size_mib(self):
+        """Calculates the model size from HF accelerate
+
+        This function gets the model size from accelerate. It also adds a
+        padding and converts to size MiB. When performing inference, expect
+        to add up to an additional 20% to the given model size as found by EleutherAI.
+        """
+        dtypes = "float32"
+        try:
+            if self.env_vars.get("dtypes"):
+                dtypes = self.env_vars.get("dtypes")
+
+            parser = estimate_command_parser()
+            args = parser.parse_args([self.model, "--dtypes", dtypes])
+        except Error:
+            logging.error("Args specified incorrect for model %s", self.model)
+
+        output = gather_data(args)  # "dtype", "Largest Layer", "Total Size Bytes", "Training using Adam"
+
+        total_memory_size_mib = 1.2 * output[0][2] * 0.00000095367431640625
```

Comment:
nit: 1.2 and 0.00000095367431640625 are magic numbers, in the official PR lets set them as constants and have comments stating what each do

Reply:
thanks, will do